dockerfile/anaconda-iso-creator-webui/lorax-build

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Build a boot.iso by lorax. The boot.iso will be stored in the `/images/` directory.
+# We have to build the RPMs files of Anaconda first and then add them as volume
+# mount to /anaconda-rpms to the container (could be RO mount).
+#
+#   make -f ./Makefile.am container-rpms-scratch
+#
+# Input directory:
+# /anaconda-rpms/ (Anaconda RPM files for the build)
+#
+# Output directory:
+# /images (Where the boot.iso will be stored)
+#
+
+set -eux
+
+INPUT_RPMS=/anaconda-rpms/
+REPO_DIR=/tmp/anaconda-rpms/
+
+# create repo from provided Anaconda RPMs
+mkdir -p $REPO_DIR
+cp -a $INPUT_RPMS/* $REPO_DIR || echo "RPM files can't be copied!"  # We could just do the build with official repositories only
+createrepo_c $REPO_DIR
+
+# clone Lorax fork with modified templates, which manipulate BIOS & EFI boot options to boot into the Web UI
+git clone --branch master-webui_boot_options https://github.com/M4rtinK/lorax/ /tmp/lorax
```

Comment:
Theoretically we can solve these changes by an environment variable. If there will be invocation with `-e "WEBUI=1"` as the podman call parameter then we don't need any of this duplication.